<launch>

  <!-- Camera parameters -->
  <arg name="camera_name"        default="/camera/color" />
  <arg name="camera_frame"       default="camera" />
  <arg name="image_topic"        default="/image_raw" />
  <arg name="publish_image"      default="true" />

  <node pkg="work_vision"
        type="container_detector_red_node.py"
        name="container_detector_red_node"
        output="screen">
  </node>


</launch>